libjcat/jcat-self-test.c

```diff
@@ -1679,151 +1679,465 @@ jcat_rfc6962_func2(void)
 	g_assert_cmpint(proof_right->len, ==, 5);
 }
 
-#if 0
-inclusionProofTestVector struct {
-	     gint64 leaf;
-	 gint64 snapshot;
-	    GPtrArray *proof;
+struct inclusionProofTestVector {
+	gint64 leaf;
+	gint64 snapshot;
+	GPtrArray *proof;
+};
```

Comment:
for structs like this, can you please do:

    typedef struct {
        the-stuff-inside
    } inclusionProofTestVector;

then you can convert the `struct inclusionProofTestVector` to `inclusionProofTestVector` in all the code below.

Reply:
Done.